hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/fee/CryptoFeeBuilder.java

```diff
@@ -118,7 +118,7 @@ public FeeData getCryptoDeleteTxFeeMatrices(final TransactionBody txBody, final
      * @param crCreateSize the size of the crypto create transaction body
      * @return the total RAM Bytes for the given crypto create transaction body
      */
-    private long getCryptoRBS(final CryptoCreateTransactionBody cryptoCreate, final int crCreateSize) {
+    private static long getCryptoRBS(final CryptoCreateTransactionBody cryptoCreate, final int crCreateSize) {
```

Comment:
It appears that all of the public methods on this class should probably be static.
Side note, that suggests this class, and its parent, represent a stateless Factory, rather than a stateful Builder.

Reply:
yeah makes sense

Reply:
Done